### PR TITLE
Correctly translated into Russian

### PR DESCRIPTION
**Pull request by n1k1c4**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by greggubarev** [comment]
LGTM 👍 :dart:

Thank you for the PR ✨ 🎉
 We're happy to accept these changes and look forward to your future contributions :clap:

Happy coding! :octocat:

File: guide/russian/css/padding/index.md [hunk 1]
```diff
@@ -1,58 +1,55 @@
 ---
 title: Padding
-localeTitle: Отступ
+
+localeTitle: Внутренний отступ
 ---
-# Отступ
 
-Свойство `padding` CSS задает область заполнения на всех четырех сторонах элемента. Это свойство может использоваться для создания пространства вокруг контента (внутри границы). Это краткое описание, чтобы установить все отдельные paddings сразу: `padding-top` , `padding-right` , `padding-bottom` и `padding-left` . Значения определяются по часовой стрелке.
```
Comment (manhauze1) [inline guide/russian/css/padding/index.md]: Manh Azure

Comment (manhauze1) [reply guide/russian/css/padding/index.md]: Thanh-tra-chinh-phu

File: guide/russian/css/padding/index.md [hunk 2]
```diff
@@ -1,58 +1,55 @@
...
+# Внутренний отступ
+Свойство CSS `padding` задаёт внутренние отступы со всех четырёх сторон элемента. Это свойство можно использовать для создания пространства вокруг содержимого (внутри границ). Это сокращённая запись для того, чтобы сразу установить все индивидуальные внутренние отступы: `padding-top`, `padding-right`, `padding-bottom` и `padding-left`. Значения определяются по часовой стрелке. Можно легко запомнить аббревиатуру T - top, R - right, B - bottom, L - left (TRBL), как слово `trouble`.
```
Comment (manhauze1) [inline guide/russian/css/padding/index.md]: Manh Azure

File: guide/russian/css/padding/index.md [hunk 3]
```diff
@@ -1,58 +1,55 @@
...
+Значения внутреннего отступа устанавливаются с помощью длины или процентов или ключевого слова `inherit` и не могут принимать отрицательные значения. Начальное значение или значение по умолчанию для всех свойств внутреннего отступа равно 0. В то время как вы можете использовать ключевое слово `inherit`, но оно не может быть использовано вместе со значением длины.
 
 ## Синтаксис
-
 ```css
-.element { 
-    padding: [padding-top] || [padding-right] || [padding-bottom] || [padding-left]; 
- } 
+.element {
+    padding: [padding-top] || [padding-right] || [padding-bottom] || [padding-left];
```
Comment (manhauze1) [inline guide/russian/css/padding/index.md]: Manh Azure

**Review by manhauze1 — COMMENTED** [review]
Manh Azure

**Pull request merged**